core/src/main/java/org/eclipse/hono/config/ServerConfig.java

```diff
@@ -0,0 +1,209 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.config;
+
+import java.util.Objects;
+
+import org.eclipse.hono.util.Constants;
+
+/**
+ * Configuration of common properties that are valid for a health check server.
```

Comment:
> ... that are common for components accepting requests/connections on a network socket.

Reply:
Thanks, I missed that when I changed the name.